static/Makefile

```diff
@@ -9,20 +9,17 @@ HASH_CMD=docker run -v $(CURDIR):/sum -w /sum debian:jessie bash hash_files
 DIR_TO_HASH:=build/linux
 DOCKER_CLI_GOLANG_IMG=golang:$(GO_VERSION)
 
-DOCKER_BUILD_OPTS=
+BAKE_SET_FLAGS=
```

Comment:
The only thing I had with this was that I was wondering if someone may be depending on this variable; wondering if there's a way to keep it for backward-compatibility (might be tricky as it needs a different format for `bake`).

Reply:
I think we can keep the build args we have previously, let me take a look.